### PR TITLE
One Terminal Script

### PR DESCRIPTION
**Pull request by mashrukk**
## Checklist
- [x] My code adheres to the coding and style guidelines of the project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

**Comment by nnyyxxxx** [comment]
Chris said on stream that he DOES NOT want scripts being combined, this is bad and should not be done, PLEASE CLOSE THIS!!!

**Comment by mashrukk** [comment]
Really? I wasn't watching the stream, let me check

**Comment by nnyyxxxx** [comment]
> Really? I wasn't watching the stream, let me check the vod

yes

File: tabs/applications-setup/tab_data.toml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 name = "Applications Setup"
 
 [[data]]
-name = "Alacritty"
-script = "alacritty-setup.sh"
+name = "Terminals"
```
Comment (adamperkowski) [inline tabs/applications-setup/tab_data.toml:4]:
```suggestion
name = "Terminal Emulators"
```

File: tabs/applications-setup/terminal-setup.sh [hunk 2]
```diff
@@ -0,0 +1,83 @@
+#!/bin/sh -e
+
+. ../common-script.sh
+
+chooseTerminal() {
+    echo "Choose your preferred terminal:"
+    echo "1) Alacritty"
+    echo "2) Kitty"
+    echo "Enter your choice (1-2): "
```
Comment (adamperkowski) [inline tabs/applications-setup/terminal-setup.sh:9]:
```suggestion
    printf "Choose your preferred terminal emulator:\n"
    printf "1) Alacritty\n"
    printf "2) Kitty\n"
    printf "Enter your choice (1-2): "
```

File: tabs/applications-setup/terminal-setup.sh [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+            printf "%b\n" "${RED}Invalid choice. Please choose one of the options available.${RC}"
+            exit 1
+            ;;
+    esac
+}
+
+installAlacritty() {
+    printf "%b\n" "${YELLOW}Installing Alacritty...${RC}"
+    if ! command_exists alacritty; then
+        case "$PACKAGER" in
```
Comment (adamperkowski) [inline tabs/applications-setup/terminal-setup.sh:31]:
```suggestion
    if ! command_exists alacritty; then
        printf "%b\n" "${YELLOW}Installing Alacritty...${RC}"
        case "$PACKAGER" in
```

File: tabs/applications-setup/terminal-setup.sh [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+                $ESCALATION_TOOL "$PACKAGER" install -y alacritty
+                ;;
+        esac
+    else
+        printf "%b\n" "${GREEN}Alacritty is already installed.${RC}"
+    fi
+}
+
+setupAlacrittyConfig() {
+    printf "%b\n" "${YELLOW}Copy alacritty config files${RC}"
```
Comment (adamperkowski) [inline tabs/applications-setup/terminal-setup.sh:45]:
```suggestion
    printf "%b\n" "${YELLOW}Copying alacritty configuration files...${RC}"
```

File: tabs/applications-setup/terminal-setup.sh [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+    mkdir -p "${HOME}/.config/alacritty/"
+    curl -sSLo "${HOME}/.config/alacritty/alacritty.toml" "https://github.com/ChrisTitusTech/dwm-titus/raw/main/config/alacritty/alacritty.toml"
+    curl -sSLo "${HOME}/.config/alacritty/nordic.toml" "https://github.com/ChrisTitusTech/dwm-titus/raw/main/config/alacritty/nordic.toml"
+    printf "%b\n" "${GREEN}Alacritty configuration files copied.${RC}"
+}
+
+installKitty() {
+    printf "%b\n" "${YELLOW}Install Kitty if not already installed...${RC}"
+    if ! command_exists kitty; then
+        case "$PACKAGER" in
```
Comment (adamperkowski) [inline tabs/applications-setup/terminal-setup.sh:58]:
```suggestion
    if ! command_exists kitty; then
        printf "%b\n" "${YELLOW}Installing Kitty...${RC}"
        case "$PACKAGER" in
```

**Review by adamperkowski — CHANGES_REQUESTED** [review]

**Comment by nnyyxxxx** [comment]
@adamperkowski don't bother, this won't be merged anyways

**Comment by mashrukk** [comment]
@nnyyxxxx I see, since he rejected your PR #382 . I don't think he'll like this either.
image

**Comment by adamperkowski** [comment]
> don't bother, this won't be merged anyways

I know. I just saw your messages after reviewing. Quite unfortunate.

**Comment by nnyyxxxx** [comment]
@mashrukk this should be closed..

**Comment by mashrukk** [comment]
@nnyyxxxx I see, since he rejected your PR #382 . I don't think he'll like this either. I'm gonna still put it up. If he closes, he will
image

**Comment by adamperkowski** [comment]
> I see, since he rejected your PR https://github.com/ChrisTitusTech/linutil/pull/382 . I don't think he'll like this either. I'm gonna still put it up. If he closes, he will

I'm sorry but if you know this is going to be closed eventually, you're just wasting Chris' and our time now.

**Comment by mashrukk** [comment]
Alright then

**Pull request closed without merging**